netbout-web/src/test/java/com/netbout/rest/account/TkSaveEmailTest.java

```diff
@@ -47,6 +47,10 @@
  * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
  */
 public final class TkSaveEmailTest {
+    /**
+     * Email parameter name.
+     */
+    private static final transient String EMAIL = "email";
```

Comment:
@mkordas `EMAIL` is the name of  the API parameter so it has the meaning in real world

>Constants (as with any object in OOP) should have a meaning in a real world.